### PR TITLE
fixing set_named_target method

### PR DESCRIPTION
**Pull request by georgiablanco**
## Proposed changes

adding a try except for  set_named_target method for the bug: https://shadowrobot.atlassian.net/browse/SRC-5834

this was fixed as recommendation seen here: https://github.com/Kinovarobotics/kinova-ros/issues/110

## Checklist

If the task is applicable to this pull request (see applicability criteria in brackets), make sure it is completed before checking the corresponding box. Otherwise, tick the box right away. Make sure that **ALL** boxes are checked **BEFORE** the PR is merged.

- [x] Manually tested that added code works as intended (if any functional/runnable code is added).
- [x] Added automated tests (if a new class is added (Python or C++), interface of that class must be unit tested).
- [x] Tested on real hardware (if the changed or added code is used with the real hardware).
- [x] Added documentation (For any new feature, explain what it does and how to use it. Write the documentation in a relevant space, e.g. Github, Confluence, etc.)

### CODE REVIEW

File: sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py [hunk 1]
```diff
@@ -335,7 +335,10 @@ def set_named_target(self, name):
             for n, this_name in enumerate(joints):
                 if this_name in active_names:
                     js[this_name] = positions[n]
-            self._move_group_commander.set_joint_value_target(js)
+            try:
+                self._move_group_commander.set_joint_value_target(js)
+            except Exception as e:
+                print(e)
```
Comment (mykolasjuraitis) [inline sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py]: Why not `rospy.logerr`?

**Review by mykolasjuraitis — COMMENTED** [review]
I think today in #pr-reviews channel we decided that all checkboxes must be checked to say that even if you didn't add new tests/etc. you still considered it and decided not to do it.

**Review by ghost — APPROVED** [review]

File: sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py (hunk 1 above)
Comment (georgiablanco) [reply sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py]: sorry yes changing now was debugging too quick haha. will do rosinfo as it isn't really an error?

**Review by ghost — APPROVED** [review]

File: sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py (hunk 1 above)
Comment (mykolasjuraitis) [reply sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py]: ok

**Review by mykolasjuraitis — APPROVED** [review]

**Pull request merged**